app/components/public/speaker-item.js

```diff
@@ -1,7 +1,12 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed } = Ember;
 
 export default Component.extend({
-  classNames: ['column', 'four', 'wide']
+  classNameBindings: ['style'],
+
+  style: computed(function() {
```

Comment:
Ahh missed one thing. You need to specify computed to observe a variable for changes. Here, we need `style` to be recomputed every time `isFeatured` changes. So, you need to specify the name of the variable to observer as the first argument of `computed`. You can specify multiple variables as multiple arguments.